Adding content and warning to the README.md (#754)
MicrosoftDocs/quantum-docs-pr: README.md

# Microsoft Quantum Documenation
Welcome to the official repository for the Microsoft Quantum Development Kit. 

This repository contains open-source documentation articles and automatically generated API files. There are two main folders:
- **[articles/](./articles)**: this folder contains the Markdown articles and YAML files that constitute the source of [the live version of the documentation](https://docs.microsoft.com/quantum/?view=qsharp-preview).
- **[api/](./api)**: this folder contains the automatically generated API documentation from the original QDK source code comments that constitue the source of the live version of the [API reference](https://docs.microsoft.com/en-us/qsharp/api/?view=qsharp-preview).

> > :warning: If you want to modify an article of the API reference DO NOT modify the `*.yml` file in this repository. These files are automatically generated from the comments in the source code files of the repositories: [Quantum Libraries](https://github.com/microsoft/QuantumLibraries) and [Q# runtime](https://github.com/microsoft/qsharp-runtime). This means that any change made in those `*.yml` files will be overwritten in the next build. To modify the article you should go to the original source file and apply the changes in the comments there.

Reviewer: ```suggestion
> :warning: If you want to modify an article of the API reference, **DO NOT** modify the `*.yml` file in this repository. These files are automatically generated from the comments in the source code files of the repositories: [Quantum Libraries](https://github.com/microsoft/QuantumLibraries) and [Q# runtime](https://github.com/microsoft/qsharp-runtime). This means that any changes made in those `*.yml` files will be overwritten in the next build. To modify the article you should go to the original source file and apply the changes in the comments there.
> For more details, please see the [documentation section of the contribution guide](https://docs.microsoft.com/quantum/contributing/docs).
```

# Microsoft Quantum Documenation

> Welcome to the official repository for the Microsoft Quantum Development Kit. 

Reviewer 2: ```suggestion
Welcome to the official documentation repository for the Microsoft Quantum Development Kit. 
```

Reviewer 2: Maybe?
```suggestion
Welcome to the official documentation repository for Microsoft's Quantum Development Kit. 
```

This repository contains open-source documentation articles and automatically generated API files. There are two main folders:

> - **[articles/](./articles)**: this folder contains the Markdown articles and YAML files that constitute the source of [the live version of the documentation](https://docs.microsoft.com/quantum/?view=qsharp-preview).

Reviewer 2: ```suggestion
- **[articles/](./articles)** contains the Markdown articles and YAML files that constitute the source of the [QDK documentation](https://docs.microsoft.com/quantum/?view=qsharp-preview).
```

> - **[api/](./api)**: this folder contains the automatically generated API documentation from the original QDK source code comments that constitue the source of the live version of the [API reference](https://docs.microsoft.com/en-us/qsharp/api/?view=qsharp-preview).

Reviewer 2: ```suggestion
- **[api/](./api)** contains the *automatically generated* API documentation that serves as the source for the [Q# API reference](https://docs.microsoft.com/en-us/qsharp/api/?view=qsharp-preview).
```

> > :warning: If you want to modify an article of the API reference, **DO NOT** modify the `*.yml` file in this repository. These files are automatically generated from the comments in the source code files of the repositories: [Quantum Libraries](https://github.com/microsoft/QuantumLibraries) and [Q# runtime](https://github.com/microsoft/qsharp-runtime). This means that any changes made in those `*.yml` files will be overwritten in the next build. To modify the article you should go to the original source file and apply the changes in the comments there.

Reviewer 2: ```suggestion
> :warning: If you want to modify an article of the API reference, **DO NOT** modify the `*.yml` file in this repository, as any changes would be automatically overwritten in the next build. These files are automatically generated from the comments in the language source code files, which are located in the repositories [Quantum Libraries](https://github.com/microsoft/QuantumLibraries) and [Q# runtime](https://github.com/microsoft/qsharp-runtime). To modify the article you should locate the original source file, apply the changes in the comments there, and submit the pull request to its respective repository.
```
